apps/condo/domains/common/components/containers/FormList.tsx

```diff
@@ -410,7 +410,19 @@ const FormWithAction: FunctionComponent<IFormWithAction> = (props) => {
         //}
     }
 
-    function handleChange (changedValues, allValues) {
+    const errors = {}
+
+    const throttledValidateFields = throttle((field) => {
```

Comment:
why throttle? and no debounce?

Reply:
https://css-tricks.com/debouncing-throttling-explained-examples/

Reply:
`debounce` will **not** call the validator during the input process, only in the end, but the `throttle` will